Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-image-gallery/index.jsx

```diff
@@ -57,7 +57,7 @@ const CloudinaryImageGallery = ({size, cloudinaryImageGallery = {}}) => {
                         <>
                             <Box {...styles.heroImageGroup}>
                                 <AspectRatio {...styles.heroImage} ratio={1}>
-                                    <DynamicImage src={`${imageUrl.url}[?sw={width}&q=60]`} />
+                                    <DynamicImage src={`${imageUrl.url.lastIndexOf('?') > -1 ? imageUrl.url.substring(0, imageUrl.url.lastIndexOf('?')) + '?_i=AP3' : imageUrl.url + '?_i=AP3'}[?sw={width}&q=60]`} />
```

Comment:
We should use a common function to convert the analytics tag, and to possibly convert the SFCC image manipulation, e.g. [?sw={width}&q=60]

Reply:
If we use a separate function then it first renders the URL with the tag coming from int_cloudinary, we have resolved this by directly replacing the tag within the src, by doing this only append _i=AP3 tag